src/routes/+page.svelte

```diff
@@ -29,7 +33,17 @@
 				<CategoryChooserView />
 				<WishListButtonView />
 			</div>
-			<TextInput bind:value={$searchFilter} placeholder={$t('Input.Search')} />
+			<div class='relative'>
```

Comment:
Looks snazzy!

A few things to change before it can go out:
- Move this into its own view called `SearchView`
- Increase the `stroke-width` of the magnifying glass to `2` and move it into the `icons/` folder
- Use the `x-mark` icon for the X